src/riscv/lib/src/jit/state_access.rs

```diff
@@ -292,12 +292,12 @@ extern "C" fn f64_from_x64_unsigned_dynamic<MC: MemoryConfig>(
     core: &mut MachineCoreState<MC, Owned>,
     exception_out: &mut MaybeUninit<Exception>,
     xval: XValue,
-    f64_out: &mut MaybeUninit<f64>,
+    f64_out: &mut MaybeUninit<FValue>,
```

Comment:
`fval_out` instead?

Reply:
Changed [here](https://github.com/tezos/riscv-pvm/pull/157/files#diff-3efb6e3ea99258cc4124ee9f32f9d516d38f427d2f053d300a10f3097358182eL295-R294)

Reply:
sorry, I wasn't clear - I meant fval_out for the variable name would be clearer I think 
```suggestion
    fval_out: &mut MaybeUninit<FValue>,
```